### DISCUSSION TITLE
Tile list not showing and "GO" button not working on Samsung Family Hub browser

### DISCUSSION THREAD

**Discussion by skydado** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#664] [Q&A]
I succesfully integrated my Dreame Z10 Pro with HA and set up a fully working xiaomi-vacuum-map-card with extracted map.
All works perfectly on my PC browser and HA Companion app.

I'd like to use it on my Samsung Family Hub integrated tablet as I do for all other dashboards and cards.
Unfortunatly, on that tablet, the tile list is not showed at all and the "go" (play) button (next to x1) does nothing.
All other buttons work perfectly: zone chooser, the "+" to add a custom zone, zoom, etc

I know it is a limit of the browser used in that browser (I can't change it or updated it), but I'm wondering if there is a simple solution, due to the fact that all other cards are working fine and part of this card is also working as expected.

Please find attached details about the browser and a picture that shows how the card shoes on that.

![xiaomi-vacuum-map-card](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/144361321/55a0cfa8-8981-4916-8113-eb4eee09931f)

![Samsung Family Hub browser details](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/144361321/163d5307-ae44-43ad-a6ad-afd903c7802a)

**Comment by PiotrMachowski** [comment]
Related issue: https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/issues/411

It's hard to fix it because I'm not really sure what's causing these issues :(

Do tiles work when you use a card with static config? (Create a copy of your config before generating static config)

**Comment by skydado** [comment]
I did't find previous topic, sorry for that.

Yes, tiles work with static config! That's an acceptable workaround. Don't know if this allows you to fix the dynamic config too ... I hope so.

I wanted to give #411 solution a try but I guess file structure is changed meanwhile.
I'm not an expert though, so I tried to replace "replaceAll" with the new syntax, but I only found 2 out of 4 changes in the following file: 

`/config/www/community/lovelace-xiaomi-vacuum-map-card/xiaomi-vacuum-map-card.js`:
```
            .replaceAll("[", "")
            .replaceAll("]", "")
            .replaceAll('"', "")
```
and
```
            .replaceAll(",", ", ")
```
.... but maybe I'm not looking in the right place.
Can you please give me detailed instructions?

If useful, if I push the GO button without selecting anything, I see a "no selection" (in Italian: "nessuna selezione") message for a couple of seconds on the Tizen/browser screen, while if I do it after zone/room selection, it does nothing. So button is working somehow, it seems more a problem of focus peraphs.

**Reply by PiotrMachowski** [reply]
> Yes, tiles work with static config! That's an acceptable workaround. Don't know if this allows you to fix the dynamic config too ... I hope so.

It's a good hint 👍

**Reply by PiotrMachowski** [reply]
Can you check the result of this test?
https://jsfiddle.net/xwj6v3np/

**Reply by skydado** [reply]
here it is

![Immagine WhatsApp 2023-10-09 ore 14 33 57_8248d0d3](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/144361321/f2c3548c-f582-4f8c-90d5-4c334da5b282)

**Reply by PiotrMachowski** [reply]
Great, thank you!

**Reply by skydado** [reply]
Have you found the mystery behind the Tizen?
May I test something?
It's really frustating to have all working except the GO button ... :(

**Reply by PiotrMachowski** [reply]
@skydado unfortunately you have to patient, I don't have much free time recently
